README.md

```diff
@@ -13,38 +12,31 @@ Speech data acquired from [Kaggle's speech accent archive](https://www.kaggle.co
 - Slow speech: hungarian8.wav
 - Thick accent: bambara4.wav
 - Normal speech: serbian9.wav
+An exerpt of Trump speaking on COVID-19 was used a real life example
+An exposition of the book of Philippians was used as a lengthy example (approx. 9 minutes)
 
 ## Engines
-- IBM Watson
-- Dialogflow
 - CMUSphinx
 - Google Speech Recognition
-- Kaldi
-- Houndify
```

Comment:
oh, I see.

You could mention you tried some of the others but they were difficult to implement, just as an "I tried them btw not news to me"